src/Bugsnag/Polyfills.cs

```diff
@@ -13,7 +13,15 @@ public static IEnumerable<string> ReadLines(string file)
 #if NET35
       return Enumerable.Empty<string>();
 #else
-      return File.ReadLines(file);
+      try
+      {
+        return File.ReadLines(file);
+      }
+      catch (Exception exception)
```

Comment:
```suggestion
      catch (System.Exception exception)
```

Reply:
I noticed that, but across the repo it's already inconsistent and if you've got a `using` clause it is unnecessary to qualify (Resharper, for example, would warn on that).  I considered changing everything in the repo to match, but thought that lightest touch was best at present.

Reply:
No strong view, although I would say that for our codebase Exceptions have more than one meaning.

Reply:
Yup, I can see the logic in that.